tests/StorageTestDefs.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * GPII Storage Integration Test Definitions
+ *
+ * Copyright 2017 OCAD University
```

Comment:
Assign copyright to whatever agency employs Astea - RtF-I?

Reply:
We have a JIRA ticket for updating the files' header information: https://issues.gpii.net/browse/GPII-3045. Are there any more modifications that we need to make to this text? Once we have the final version, we can go ahead and change the text in our code.

Reply:
Looks like Gregg has provided a final wording for these headers on the JIRA so please go ahead with that

Reply:
We believe it would be better if we open a separate PR for this task. Do you think it is something that can be postponed?

Reply:
Absolutely, let's do this in a further PR